### DISCUSSION TITLE
Comment input textbox

### DISCUSSION THREAD

**Discussion by mirabilos** [community/community#19646] [Projects and Issues]
When I enter a list (`- something` + Enter), it is rather slow (too much JS run?) **and** automatically adds `- ` on the start of the next line, totally breaking my workflow of entering these myself, especially bad when shifting indentation levels a lot.

Please show me where I can disable this so that the comment input field is just a simple `<textarea>` with no fancy JS.

**Comment by benjiallen** [comment]
Hi @mirabilos, I'm trying to understand if this is an accessibility issue or not. Would you mind sharing whether you use assistive technology or have a disability which makes this task particularly difficult?

If not, no worries, I'll just move this feedback to another category.
